app/src/ui/main-process-proxy.ts

```diff
@@ -90,7 +91,7 @@ export const focusWindow = sendProxy('focus-window', 0)
 const _showItemInFolder = invokeProxy('show-item-in-folder', 1)
 
 export const showItemInFolder = (path: string) =>
-  stat(path)
+  pathExists(path)
     .then(() => _showItemInFolder(path))
     .catch(err => log.error(`Unable show item in folder '${path}'`, err))
```

Comment:
Wouldn't this catch handler never be called, since the result is being passed through a boolean in the fulfilled handler instead?

Reply:
You're correct that `pathExists` doesn't throw. `_showItemInFolder` could still throw though and its exception would travel over the IPC boundary and be caught here.